src/NormalizedCollection/libs/RecordSet.js

```diff
@@ -74,13 +74,16 @@ util.inherits(RecordSet, AbstractRecord, {
    * @returns {Object}
    */
   mergeData: function(snaps, isExport) {
-    var self = this;
-    var out = {};
-    util.each(snaps, function(snap) {
-      if( snap.val() !== null && self.filters.test(snap.val(), snap.key(), snap.getPriority()) ) {
-        out[snap.key()] = isExport? snap.exportVal() : snap.val();
-      }
-    });
+    var self = this, out = null;
```

Comment:
This fixes an inconsistency with the Firebase SDK. When the value is `null` firebase-util was incorrectly returning an object with `{.value, .priority}` but Firebase only returns null (there is no priority for a null value).